docs/source/features.rst

```diff
@@ -267,6 +267,59 @@ Done! This particular issue will no longer show up in your scan results.
 Limiting Scans by Path
 ++++++++++++++++++++++
 
+By default ``tartufo`` will scan all objects tracked by Git. You can limit
+scanning by either including fewer paths or excluding some of them using
+Python Regular Expressions (regex).
+
+.. warning::
+
+   Using include patterns is more dangerous, since it's easy to miss the
+   creation of new secrets if future files don't match an existing include
+   rule. We recommend only using fine-grained exclude patterns instead.
+
+.. code-block:: toml
+
+   [tool.tartufo]
+   include-path-patterns = [
+      'src/',
+      'gradle/',
+      # regexes must match the entire path, but can use python's regex syntax
+      # for case-insensitive matching and other advanced options
+      '(.*/)?id_[rd]sa$',
+      # Single quoted strings in TOML don't require escapes for `\` in regexes
+      '(?i).*\.(properties|conf|ini|txt|y(a)?ml)$',
+   ]
+   exclude-path-patterns = [
+      '(.*/)?\.classpath$',
+      '.*\.jmx$',
+      '(.*/)?test/(.*/)?resources/',
+   ]
+
+The filter expressions can also be specified as command line arguments.
+Patterns specified like this are merged with any patterns specified
+in the config file:
+
+.. code-block:: sh
+
+   # --include-path-patterns == -ip
+   # --exclude-path-patterns == -xp
+   > tartufo -ip 'src/' -ip 'gradle/' \
+     -xp '(.*/)?\.classpath$' -xp '.*\.jmx$' \
+     scan-local-repo file://path/to/my/repo.git
```

Comment:
For the purposes of documentation, I think it's best to be fully explicit. Especially since you're having to call these out above.

I also added an extra line break, just for readability.

```suggestion
   > tartufo \
     --include-path-patterns 'src/' -ip 'gradle/' \
     --exclude-path-patterns '(.*/)?\.classpath$' -xp '.*\.jmx$' \
     scan-local-repo file://path/to/my/repo.git
```